### DISCUSSION TITLE
How to show the types as comments?

### DISCUSSION THREAD

**Discussion by TomzBench** [mrcjkb/rustaceanvim#220] [Q&A] [answered]
Hello,

I'm coming from the now deprecated rust-tools and installation went pretty smooth. However, I used to have the Types rendered all over my IDE that Ive kind of miss now.  I haven't found the setting to enable them.  Is this feature available?

**Comment by mrcjkb** [comment, marked as answer]
Hey 👋 

See [this discussion](https://github.com/mrcjkb/rustaceanvim/discussions/46).
